libglusterfs/src/iobuf.c

```diff
@@ -594,9 +650,19 @@ iobuf_get(struct iobuf_pool *iobuf_pool)
 {
     struct iobuf *iobuf = NULL;
     int index = 0;
+    size_t page_size = 0;
 
     GF_VALIDATE_OR_GOTO("iobuf", iobuf_pool, out);
 
+    page_size = iobuf_pool->default_page_size;
+    if (page_size <= USE_IOBUF_POOL_IF_SIZE_GREATER_THAN) {
+        iobuf = iobuf_get_from_small(page_size);
+        if (!iobuf)
+            gf_smsg(THIS->name, GF_LOG_WARNING, 0, LG_MSG_IOBUF_NOT_FOUND,
+                    NULL);
+        return iobuf;
+    }
+
```

Comment:
Why we don't replace this entire function by a call to `iobuf_get2()` with the default page size of the pool ?

Reply:
done